docs/guides/administrator/adding-and-managing-users.md

```diff
@@ -0,0 +1,24 @@
+# Adding and Managing Users
+
+## Introduction
+
+An administrator can provision and manage users in Opstrace.
+Users allow authentication against our web UI and the per-tenant Grafana UI.
```

Comment:
Let's maybe reword "Users allow authentication"

Sounds like there is a group of people that allow me to do something :-).

A key difference between tenants and users that we should point out somewhere:

- A tenant's identity is meant to be used by machines/scripts.
- A user's identity is meant to be used by human users sitting in front of the browser, never by scripts.

Reply:
> we should point out somewhere

Good point.  Perhaps we should move it to the key concepts?  I'll do that in a separate PR.

In the meantime I will modify the sentence with your suggestions.